### DISCUSSION TITLE
Access Mikrotik devices with SSH KEY

### DISCUSSION THREAD

**Discussion by kalazino** [eduardogsilva/routerfleet#42] [Q&A]
HI,

despite having generated the SSH keys on the Mikrotik, and importing them into routerfleet, I am unable to add the device.
Is there any bug?

Thank you

**Comment by eduardogsilva** [comment]
Hello, there's no know issue with this feature. 

Please, make sure that you can login properly to your router using ssh key authentication.

Cheers!

**Reply by kalazino** [reply]
Hello and thanks for the reply.
The procedure I followed is this:
- generation and export of the mikrotik host key
- on the mikrotik, imported private key and combined with user management

On routerfleet:
- copied private key and public key

If I log in from another device using the key, it works normally

A thousand thanks
